lib/tasks/one_offs.rake

```diff
@@ -1,11 +1,11 @@
 # frozen_string_literal: true
 
 namespace :one_offs do
-  desc "Extend ECF Standard April started milestone date"
-  task extend_ecf_standard_april_started_milestone_date: :environment do
-    schedule = Finance::Schedule.find_by(name: "ECF Standard April")
-    schedule.schedule_milestones.find_by(name: "Output 1.1 - Participant Start").destroy!
-    Finance::Milestone.find_by(name: "Output 1.1 - Participant Start").destroy!
-    schedule.milestones.find_by(declaration_type: "started").update!(milestone_date: Date.new(2022, 5, 31))
+  desc "backfill teacher catchement iso country code"
+  task backfill_teacher_catchment_iso_country_code: :environment do
+    NPQApplication.where.not(teacher_catchment_country: nil).find_each do |npq_application|
+      country = ISO3166::Country.find_country_by_iso_short_name(npq_application.teacher_catchment_country)
```

Comment:
Again it would be interesting to know which of these don't match, but that's easy enough to do afterwards. There are some funny ones in the gem that might not cleanly match, like `Bolivia (Plurinational State of)`.